.travis.yml

```diff
@@ -1,26 +1,30 @@
 os:
   - linux
-dist: trusty
+dist: xenial
 
 language: c
 
 cache:
   directories:
     - "$HOME/.choosenim"
 
+env:
+  global:
+    - CHOOSENIM_CHOOSE_VERSION="0.20.0"
+    - CHOOSENIM_NO_ANALYTICS=1
+    - PATH=$HOME/.nimble/bin:$PATH
+
 install:
-  - export CHOOSENIM_CHOOSE_VERSION="0.18.0"
   - |
       curl https://nim-lang.org/choosenim/init.sh -sSf > init.sh
       sh init.sh -y
-  - export PATH=$HOME/.nimble/bin:$PATH
 
 before_script:
   - set -e
   - set -x
 
 script:
-  - nim c -d:ssl -r package_scanner.nim && node ./validate_json.js
+  - nim c -d:ssl -d:release -r package_scanner.nim
```

Comment:
-d:release slows down the compile time and does not improve significantly the tool run speed